src/torchmetrics/functional/text/helper_embedding_metric.py

```diff
@@ -23,13 +23,11 @@
 from torchmetrics.utilities.data import _cumsum
 from torchmetrics.utilities.imports import _TQDM_AVAILABLE, _TRANSFORMERS_GREATER_EQUAL_4_4
 
-if _TRANSFORMERS_GREATER_EQUAL_4_4:
-    from transformers import AutoModelForMaskedLM, AutoTokenizer, PreTrainedModel, PreTrainedTokenizerBase
-else:
-    PreTrainedModel = PreTrainedTokenizerBase = None
-
-if _TQDM_AVAILABLE:
-    import tqdm
+if TYPE_CHECKING:
+    if _TQDM_AVAILABLE:
+        import tqdm
+    if _TRANSFORMERS_GREATER_EQUAL_4_4:
+        from transformers import PreTrainedModel, PreTrainedTokenizerBase
```

Comment:
AFAIK, under TYPE_CHECKING the other checks do not matter

```suggestion
    import tqdm
    from transformers import PreTrainedModel, PreTrainedTokenizerBase
```